doc/content/getting-started/installation/_index.md

```diff
@@ -2,10 +2,11 @@
 title: "Installing The Things Stack"
 description: ""
 distributions: ["Enterprise", "Open Source"]
-weight: -1
 ---
 
-This is a guide for setting up a private LoRaWAN network server using {{% tts %}} for LoRaWAN.
+This is a guide for installing {{% tts %}} Enterprise or Open Source on your own hardware, to run your own private LoRaWAN network server.
+
+To use {{% tts %}} in the cloud without installing it on your own hardware, see [{{% tts %}} Cloud]({{< relref "cloud-hosted" >}}) (our professional, SLA backed offering) or [The Things Network]({{< relref "ttn" >}}) (our free-to-use community edition).
```

Comment:
```suggestion
To use {{% tts %}} in the cloud without installing it on your own hardware, see [{{% tts %}} Cloud]({{< relref "cloud-hosted" >}}) (our professional, SLA-backed offering) or [The Things Network]({{< relref "ttn" >}}) (our free-to-use community edition).
```

Reply:
I think you should replace The Things Network with The Things Stack Community Edition, since you're talking about public community deployment and not about ecosystem/community (https://github.com/TheThingsIndustries/design/blob/master/guides/branding.md)